src/App.ts

```diff
@@ -692,12 +711,7 @@ export default class App {
     this.listeners.push([onlyCommands, matchCommandName(commandName), ...listeners] as Middleware<AnyMiddlewareArgs>[]);
   }
 
-  // matches a Slack function callback specifically
```

Comment:
Removed due to reverted commit included in history after bad rebase. The changed version should match main. To verify: [L694](https://github.com/slackapi/bolt-js/blob/237a503d1e19a8e40ca9ed130dacab1be9752d53/src/App.ts#L694)